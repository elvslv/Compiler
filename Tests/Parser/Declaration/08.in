var c, e: array[1..20, 30..1000, 27..28] of float;