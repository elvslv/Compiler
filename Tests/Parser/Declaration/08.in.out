var C : array [const 27..const 28] of array [const 30..const 1000] of array [const 1..const 20] of FLOAT;
var E : array [const 27..const 28] of array [const 30..const 1000] of array [const 1..const 20] of FLOAT;
